CreeDictionary/search_quality/analyze_results.py

```diff
@@ -57,12 +57,20 @@ class DuplicateAnnotatedSearchResult(TypedDict, total=False):
     manually approximate the relation.
     """
 
-    matched_cree: str
+    wordform_text: Optional[str]
+    # legacy, deprecated
+    matched_cree: Optional[str]
+
     lemma_wordform: PartialSerializedWordform
+    is_duplicate_of: Optional[str]
 
-    raw_suffix_tags: tuple[FSTTag, ...]
 
-    is_duplicate_of: Optional[str]
+def get_result_text(r: DuplicateAnnotatedSearchResult | SerializedSearchResult):
+    # Use the new language-independent key if available
+    if "wordform_text" in r:
+        return r["wordform_text"]
+    # Fall back to old thing
+    return r["matched_cree"]
```

Comment:
Is the idea that we can eventually take this out?

Reply:
Yeah, this is the same thing as https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/pull/727#pullrequestreview-631585858

I can remove it, but then it turns into a bigger thing with needing to update the search result sampling code used on older branches for comparison purposes.

Reply:
No need to remove it now, it's just good to know that this is supporting older code.